pkgs/top-level/pidgin-packages.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, newScope
+, pidgin
+, texlive
+, config
+}:
+
+lib.makeScope newScope (self:
+  let callPackage = self.callPackage;
+  in {
+    pidgin = callPackage ../applications/networking/instant-messengers/pidgin {
+      withOpenssl = config.pidgin.openssl or true;
+      withGnutls = config.pidgin.gnutls or false;
+      plugins = [];
```

Comment:
```suggestion
      plugins = [];
      pidginPackages = self;
```

Reply:
passed as a scope element like how it is in lua instead of passing as callPackage argument